.rubocop.yml

```diff
@@ -86,23 +86,14 @@ Style/Documentation:
   Exclude:
     - 'spec/**/*'
     - 'test/**/*'
-    - 'lib/factory_bot_rails.rb'
-    - 'lib/factory_bot_rails/definition_file_paths.rb'
-    - 'lib/factory_bot_rails/generator.rb'
-    - 'lib/factory_bot_rails/generators/non_rspec_generator.rb'
-    - 'lib/factory_bot_rails/generators/null_generator.rb'
-    - 'lib/factory_bot_rails/generators/rspec_generator.rb'
-    - 'lib/factory_bot_rails/railtie.rb'
-    - 'lib/factory_bot_rails/reloader.rb'
     - 'lib/generators/factory_bot/model/model_generator.rb'
 
 # Offense count: 2
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: compact, expanded
 Style/EmptyMethod:
-  Exclude:
-    - 'lib/factory_bot_rails/generators/null_generator.rb'
+  EnforcedStyle: expanded
```

Comment:
I have no strong preference on this one. If we are going to go with `expanded`, let's move this configuration up to the top of the file out of the "TODO" section.

Reply:
@composerinteralia 👍, moved that rule to the top: https://github.com/thoughtbot/factory_bot_rails/pull/296/commits/f872dbad7fcf6d13b9c766ee69a5471a198b0d8a
(I get weirded out seeing semicolons in ruby code, so this is the route I'd personally opt for)